docs/administration/collector-metrics-and-alerts.adoc

```diff
@@ -129,12 +129,16 @@ You can view this alerts in the OpenShift Container Platform web console.
 image::buffer-alert.png[Fired allert]
 
 === CollectorNodeDown
-
 Will be fired if collector component was offline for more than 10m
 
-=== CollectorVeryHighErrorRate
+=== ElasticsearchDeprecation
```

Comment:
@Clee2691 this PR looks good to me.  Can you prepare an issue to remove these alerts from 6.x ?  None of these exist in 6.x so it isn't relevant any longer